### DISCUSSION TITLE
Create Collection Type with same name as the default one (User)

### DISCUSSION THREAD

**Discussion by khanhtmn** [strapi/strapi#7438] [Legacy Topics]
Hi, I'm a newbie in Strapi and back-end dev here. I'm using Strapi to create an admin dashboard for an available back-end with MongoDB. 

**Problem**
We have a table name `User` in our database, and thus need to create a collection type name `User`. However, it seems like this would conflict with the pre-exising `User` collection type, which is used by Strapi by default (photo attached below)
![Screenshot from 2020-08-12 18-01-14](https://user-images.githubusercontent.com/44103802/90008074-fe386c00-dcc5-11ea-9fb4-f3790a796121.png)

**Approaches I have tried:**
- I tried to add my custom fields to the collection from the UI but it doesn't work.
- I also tried to add new API endpoint in the back-end, but got this error: `Error creating endpoint delete /users/:id: handler not found "user.deleteOne"`

Any insight to work around this problem is much appreciated. Many thanks in advance!

**Comment by derrickmehaffy** [comment]
@alexandrebodin I believe we had this as something to look into for v4 no?

**Reply by alexandrebodin** [reply]
@derrickmehaffy Except from making the plugin optional eventually (probably not in v4) not really no.

@khanhtmn What you can do right now is the following:

- Create a CollectionType named sth like **ApplicationUser** with the right attributes and set the `collectionName` to your user table name.
- Your api routes will be /application-users but this should work just fine

**Reply by khanhtmn** [reply]
Superb! This is exactly what I need. Thank you so much! Have been looking this problem in the Slack channel for a while now and glad I bring it up here! @alexandrebodin
